themesberg/flowbite
# Phone input bug fixed

### **Issue**

Phone input country code no working. https://github.com/themesberg/flowbite/issues/939

### **Pull Request Description**
Summary: This pull request addresses the issue https://github.com/themesberg/flowbite/issues/939 where, the phone input country code feature does not work as it does not let the user selects a country code and remains opened with no way to close it.

### **Changes Made**
1.Implemented proper event handling to allow users to select a country code from the dropdown.
2.Fixed the issue where the dropdown would remain open after selection.
3.Ensured that the selected country code updates the input field as expected.
4.Removed the duplicate entry for Germany from the dropdown list.

### **Steps to Test**
1.Navigate to the page with the phone input field.
2.Click the dropdown, select a country code, and verify it updates the input field and closes.
3.Open the dropdown and click outside to ensure it closes.
4.Ensure functionality works across different browsers and screen sizes.

### ** Outcome Screenshots**
https://github.com/user-attachments/assets/035a26ae-5c4b-4ba8-bbda-8840554fe5fc
